Foundation/include/Poco/Types.h

```diff
@@ -21,30 +21,12 @@
 
 
 #include "Poco/Foundation.h"
-#if defined(POCO_ENABLE_CPP11)
-#include <cstdint>
```

Comment:
I added <cstdint> because it is a header guaranteed to be available in C++11.
(http://en.cppreference.com/w/cpp/types/integer)
so it would be a cross-platform way to defined the types.